src/nestjs-back/src/users/users.service.ts

```diff
@@ -186,6 +193,11 @@ export class UsersService {
     let user: User | null = null;
     let tmpDto = {};
 
+    /* Informations */
+    if (updateUserDto.username) {
+      await this.checkUsernameIsAvailable(updateUserDto.username);
```

Comment:
wording: `isUsernameAvailable` is self descriptive

Reply:
Renamed!